client/state/index.js

```diff
@@ -109,7 +109,8 @@ const middleware = [ thunkMiddleware, noticesMiddleware ];
 if ( typeof window === 'object' ) {
 	// Browser-specific middlewares
 	middleware.push(
-		require( './analytics/middleware.js' ).analyticsMiddleware
+		require( './analytics/middleware.js' ).analyticsMiddleware,
+		require( './data-layer/wpcom-api-middleware.js' ).middleware,
```

Comment:
What was the reasoning behind only enabling this in browsers? I'm under the understanding that server-side (e.g. themes) needs to make and "await" completion of network requests.